proposals/readme.adoc

```diff
@@ -1,7 +1,41 @@
 = Proposals
 
-List of proposals and associated links
+List of proposals and associated links.
 
-. link:https://github.com/microprofile/evolution/blob/master/proposals/0001-config.md[MP-0001: Service Configuration]
+This list includes both approved proposals and proposals in the discussion stage. Those proposals which are not yet approved are marked as such.
```

Comment:
Let's go with what you have and we'll adjust as needed...  At least we have the various proposals listed.  Thanks.